icu4c/source/python/icutools/databuilder/request_types.py

```diff
@@ -296,12 +297,18 @@ def __init__(self, **kwargs):
 
     def apply_file_filter(self, filter):
         i = 0
-        while i < len(self.input_files):
-            if filter.match(self.input_files[i]):
+        while i < len(self.installed_files):
+            if filter.match(self.installed_files[i]):
                 i += 1
                 continue
```

Comment:
Wouldn't "else" be clearer than "continue"?

Reply:
Done